storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -5049,7 +5248,7 @@ static pfs_os_file_t xb_delta_open_matching_space(
       ut_a(res);
 
       xb::info() << "Renaming " << dest_space_name << " to " << tmpname
-                 << ".ibd";
+                 << EXT_IBU;
```

Comment:
This should be EXT_IBD. Refactoring/cleanup mistake

Reply:
AB